### ISSUE TITLE
Renamed third-party package from `henrylee2cn` caused error

### ISSUE THREAD

**Issue by XyLearningProgramming** [bytedance/go-tagexpr#50]
When trying to install the newest version, I got the error below:
```
go get: github.com/henrylee2cn/goutil@v0.0.0-20210127050712-89660552f6f8 updating to
        github.com/henrylee2cn/goutil@v0.0.0-20220626152529-9b7868da7b6d: parsing go.mod:
        module declares its path as: github.com/andeya/goutil
                but was required as: github.com/henrylee2cn/goutil
```

**Comment by andeya** [comment]
fixed: 5ff4a5a714d1861e15237acfb833c4d357f5d16c

**Comment by 21888** [comment]
哥们这问题还存在
